docs/INSTALL.md

```diff
@@ -242,10 +242,11 @@ _macOS:_
 
 _Windows 10:_ no action needed, should work out of the box.
 
+Make sure the Studio has at least 1920Mb max heap size in "File > Settings... > Appearance & Behavior > System Settings > Memory Settings" _before_ opening the project, otherwise it might get stuck on "Updating indexes".
+
 In Android Studio open the project in `android/` directory.
-Make sure the Studio has at least 1920Mb max heap size in "File > Settings... > Appearance & Behavior > System Settings > Memory Settings", otherwise it might get stuck on "Updating indexes".
 
-Setup a virtual device to use [emulator]((https://developer.android.com/studio/run/emulator)) ("Tools > AVD Manager") or [use a hardware device for debugging](https://developer.android.com/studio/run/device).
+Setup a virtual device to use [emulator](https://developer.android.com/studio/run/emulator) ("Tools > Device Manager") or [use a hardware device for debugging](https://developer.android.com/studio/run/device). If using an emulator, make sure to choose a system image with API Level 31 and ABI _x86_64_.
```

Comment:
>make sure to choose a system image with API Level 31

IMHO specifying it is not necessary, people might want to run it on older androids.

Reply:
Agree, but I wanted to provide a hassle-free option with current targetSdk for begginers, as for me Android Studio by default selected an x86-Api 30 emulator option, which failed to launch the app.

Reply:
Wait, why did the app fail to launch in API 30 emulator? Let's debug and fix it. OM should work everywhere from Android 5.0 and above.

Reply:
Lately I'm testing on 5.0 in the emulator - everything works.

Reply:
> Wait, why did the app fail to launch in API 30 emulator? Let's debug and fix it. OM should work everywhere from Android 5.0 and above.

Should mention that when installing Android 11 Sdk 30, x86 emulator comes as recommended; but Android 12 already recommends x86_64 emulator. 
<img width="547" alt="Screen Shot 2022-02-16 at 15 01 26" src="https://user-images.githubusercontent.com/99682377/154260894-015fb64e-31d4-4e25-8d83-db4908bfafdb.png">
The crash was about ABI x86 emulator instead of x86_64 (tried to run WebDebug x86_64 build). 
Different API versions work just fine when using correct ABI.

Reply:
Ok, so the crash was not caused by our code, but by incorrect abi.